### PR TITLE
executor: Fix show status bug

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: executor/show.go [hunk 1]
```diff
@@ -318,6 +318,13 @@ func (e *ShowExec) fetchShowStatus() error {
 		if e.GlobalScope && v.Scope == variable.ScopeSession {
 			continue
 		}
+		switch v.Value.(type) {
+		case []interface{}:
+			v.Value = fmt.Sprintf("%v", v.Value)
+		}
```
Comment (coocood) [inline executor/show.go]: `case []interface{}, nil:`

**Comment by shenli** [comment]
LGTM

**Comment by zimulala** [comment]
PTAL @coocood

File: executor/show_test.go [hunk 2]
```diff
@@ -65,4 +66,24 @@ func (s *testSuite) TestShow(c *C) {
...
+	variable.RegisterStatistics(ss)
+	testSQL = "show status like 'character_set_results';"
+	result = tk.MustQuery(testSQL)
+	c.Check(result.Rows(), NotNil)
+}
+
+type statistics struct {
+}
+
+func (s statistics) GetScope(status string) variable.ScopeFlag { return variable.DefaultScopeFlag }
```
Comment (coocood) [inline executor/show_test.go]: Add an empty line.

**Comment by coocood** [comment]
LGTM

**Pull request merged**